app/services/copy_phone_configurations.rb

```diff
@@ -0,0 +1,28 @@
+class CopyPhoneConfigurations
+  def copy
```

Comment:
What do you think about naming this `call` for consistency with our other service objects?